src/utils/pgwire/src/pg_protocol.rs

```diff
@@ -513,9 +519,13 @@ where
                 .await?;
 
             // 2. Send row description.
-            self.stream
-                .write(&BeMessage::RowDescription(&statement.row_desc()))
-                .await?;
+            if statement.is_query() {
+                self.stream
```

Comment:
1. Add some comments like pg doc? 
2. define what is a query? is dml consider a query? What about other statements?

Reply:
```
let is_query_sql = {
            let lower_sql = sql.to_ascii_lowercase();
            lower_sql.starts_with("select")
                || lower_sql.starts_with("values")
                || lower_sql.starts_with("show")
                || lower_sql.starts_with("with")
                || lower_sql.starts_with("describe")
        };
```
I use this to define whether is a query sql.